pkg/filehash/cache.go

```diff
@@ -40,7 +40,7 @@ type Cache struct {
 //   - digest-inode: is the most efficient, as it keys the hash to a pair consisting of the container image digest and inode.
 //     This approach, however, necessitates container enrichment.
 func NewCache(mode config.CalcHashesOption, resolver pathResolver) (*Cache, error) {
-	hashes, err := lru.New[string, hashInfo](1024)
+	hashes, err := lru.New[string, hashInfo](10000)
```

Comment:
Does it need to be so large?

Reply:
Well, I guess it depends where we run. We should minimize as much as possible hash calculations since they are expensive, and this shouldn't take more than 1MB with this new value.
Yet, I think you are right this might be too big. I will change it to 2048.